Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -970,19 +975,38 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
 
         results = {}
 
-        # We create a file containing a semicolon separated list for the Editor to read
-        temp_batched_file = tempfile.NamedTemporaryFile(mode='w', delete=False, suffix='.txt')
-        for test_spec in test_spec_list[:-1]:
-            temp_batched_file.write(editor_utils.get_testcase_module_filepath(test_spec.test_module)
-                                    .replace('\\', '\\\\')+';')
-        # The last entry does not have a semicolon
-        temp_batched_file.write(editor_utils.get_testcase_module_filepath(test_spec_list[-1].test_module)
-                                .replace('\\', '\\\\'))
-        temp_batched_file.flush()
-        temp_batched_file.close()
+        # We create a files containing a semicolon separated scipts and test cases for the Editor to read
+        test_script_list = ""
+        test_case_list = ""
+
+        test_case_prefix = "::".join(str.split(request.node.nodeid, "::")[:2])
+        for test_spec in test_spec_list:
+            # Test script
+            test_script_list += editor_utils.get_testcase_module_filepath(test_spec.test_module) + ';'
+
+            # Test case
+            regex_result = re.search("\<class \'(.*)\'\>", str(test_spec))            
+            class_name = str.split(regex_result.group(1), ".")[-1:]
+            test_case_list += f"{'::'.join([test_case_prefix, class_name[0]])};"
```

Comment:
Would be good to pull this out into a small function, so it doesn't need to be duplicated on line 888

And even better to also write new tests for it in LyTestTools/tests/unit/